projects/1.20/assets/bygone-nether/bygonenether/lang/zh_cn.json

```
@@ -0,0 +1,111 @@
+{
+	"item.bygonenether.gilded_netherite_helmet": "镀金下界合金头盔",
+	"item.bygonenether.gilded_netherite_chestplate": "镀金下界合金胸甲",
+	"item.bygonenether.gilded_netherite_leggings": "镀金下界合金护腿",
+	"item.bygonenether.gilded_netherite_boots": "镀金下界合金靴子",

```

```suggestion
	"item.bygonenether.gilded_netherite_helmet": "镶金下界合金头盔",
	"item.bygonenether.gilded_netherite_chestplate": "镶金下界合金胸甲",
	"item.bygonenether.gilded_netherite_leggings": "镶金下界合金护腿",
	"item.bygonenether.gilded_netherite_boots": "镶金下界合金靴子",
```
统一镶金和镀金
原版有`镶金黑石`，可以看下两者有无关联